src/main/java/core/basesyntax/FigureBase.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+abstract class FigureBase implements FigureArea, FigureDrawable {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class definition.